### ISSUE TITLE
[BLOG] Green Reviews Working Group: Moving towards measuring the sustainability footprint of CNCF projects

### ISSUE THREAD

**Issue by immavalls** [cncf/tag-env-sustainability#341]
- **Authors**: @immavalls @AntonioDiTuri 
- **Reviewers**: @nikimanoledaki @rossf7 @leonardpahlke @guidemetothemoon 
- **TAG ENV Lead**: @guidemetothemoon 

## Abstract

This blog will introduce the [Green Reviews WG](https://github.com/cncf/tag-env-sustainability/blob/main/working-groups/green-reviews/charter.md), highlighting/summarizing the progress so far (e.g., https://github.com/cncf-tags/green-reviews-tooling), the milestones reached, and next steps. 

## Scope

Beginner-friendly introduction to what the WG is doing, with more technical content, what has been built so far. and where to go from here. It can also serve as a call to action for contributors that might be interested in the WG.

## Expected audience

- Beginners who have none or little knowledge about what the WG does.
- Future contributors to the WG.

## Goals and Non Goals

Goals:
- Awareness of the WG.
- Explain what the group has been working on, what is next, and how to contribute.

Non-goals:
- Get too technical or deep dive into the topics, we can point readers to the [WG's design document](https://docs.google.com/document/d/19fzZW-IMv2kDNatKFHeHh7wqcEN0e2N60wzxvCGZd48/edit?usp=sharing) that should be migrated to the WG repo. 

## Tasks

- [x] Outline Blog post
  - [x] [**Author**] described the abstract of the blog post, defined the scope, goals and non-goals, and defined the expected audience.
  - [x] [**Author**] notified TAG ENV community to review the outline via Slack on the `tag-env-sustainability` channel.
  - [x] [**TAG ENV Lead**] One of the TAG ENV leads assigned themselves to wrangle the blog
  - [x] [**TAG ENV Lead**] Blog post outline and theme has been discussed
- [x] [**Author**] Blog post written and ready for review
- [ ] Review
  - [ ] [**Author**] blog post PR opened (blog posts are added to the [`website/content/en/blog`](https://github.com/cncf/tag-env-sustainability/tree/main/website/content/en/blog) folder for English content, images to [`website/static/images/blog/<YEAR>-<MONTH>-<title>/*.jpg`](https://github.com/cncf/tag-env-sustainability/tree/main/website/static/images))
  - [ ] [**Author**] notified TAG ENV community for review via Slack on the `tag-env-sustainability` channel.
  - [ ] [**Author**] gave a minimum of a week to collect public feedback, depending on reviews this will get extended. If there is a major event (conference, etc.) or holidays coming up, please extend accordingly.
  - [ ] [**Author**] Feedback and comments have been resolved.
  - [ ] [**Reviewers**] reviewed blog post.
  - [ ] [**TAG ENV Lead**] reviewed blog post.
  - [ ] [**TAG ENV Lead**] ready to get published.
  - [ ] [**TAG ENV Lead**] notified one of the ToC liaisons to sign off the blog post.
  - [ ] [**TAG ENV ToC Liaison**] reviewed blog post.
  - [ ] [**TAG ENV Lead**] merged blog post.
  - [ ] *optional* [**TAG ENV Lead**] schedule the blog post on the CNCF blog next to the TAG ENV blog
  - [ ] [**Author**] Wrote feedback about the blog post process as a comment under this issue

## Comments
Draft document in Google Docs: https://docs.google.com/document/d/1_MnYd1hHhRysHsWCfaSuNpAEc09GuBF61Qt0rCqJDrk/edit?usp=sharing

**Comment by immavalls** [comment]
@nikimanoledaki @AntonioDiTuri @rossf7 @leonardpahlke, can you provide further details so we can start the write-up?
@guidemetothemoon can you review the issue, assign labels, possibly create the initial google doc? I am not sure I have access to do that 🙏

**Comment by leonardpahlke** [comment]
Created the Google Docs file: https://docs.google.com/document/d/1_MnYd1hHhRysHsWCfaSuNpAEc09GuBF61Qt0rCqJDrk/edit?usp=sharing

**Comment by guidemetothemoon** [comment]
Thanks a lot @immavalls for creating the issue and willing to contribute to the blog and thanks @leonardpahlke for creating the draft document. I've added labels and linked the doc to the issue description.

@nikimanoledaki @rossf7 @AntonioDiTuri please add some highlights to what you would like to be included into the post to the draft doc. Also, will one of you be able to assist Imma and others who will be writing the blog post to ensure that they get all the necessary information?

I set the deadline to KubeCon+CloudNativeCon EU but we'll see if we manage that.

@caradelia @patricia-cahill @savitharaghunathan FYI.

**Comment by immavalls** [comment]
Thanks @guidemetothemoon I created an outline in https://docs.google.com/document/d/1_MnYd1hHhRysHsWCfaSuNpAEc09GuBF61Qt0rCqJDrk/edit?usp=sharing, if you can have a look, together with @nikimanoledaki @rossf7 @AntonioDiTuri , I can start a draft. I wanted to make sure this is what we want to share in the blog or if we have different goals.

**Comment by nikimanoledaki** [comment]
Thank you all for leading this! 💚 

@immavalls, I added a few additional links to your comments in the doc. As you pointed out, there should be enough info to write a summary of the project from the Charter, Roadmap, current Milestone, docs, Community section in the README, and Contributing guide.

If you have any questions or if I can review things in a few days, please let me know. We can also grab some time to chat anytime next week and/or the week after :) 

We could also add more content as a group during the next WG Green Reviews meeting in 2 weeks on March 13th.

**Comment by immavalls** [comment]
Thanks @nikimanoledaki I will try to have an initial draft in a week so we can further discuss 🙌

**Comment by leonardpahlke** [comment]
Do we plan to release the blog before Kubecon (i suggest so - but its just 2 weeks away)? cc @caradelia @savitharaghunathan. I will assign @guidemetothemoon as TAG Lead Liaison to this blog. @nikimanoledaki @AntonioDiTuri @rossf7 could you review the content once the initial draft is ready? 

TYSM @immavalls! 🙌

**Comment by leonardpahlke** [comment]
@nikimanoledaki @AntonioDiTuri — did we already placed this issue in wg-green-reviews with a call for contribution? Perhaps someone would like to support @immavalls writing it.

**Comment by guidemetothemoon** [comment]
@leonardpahlke do you think that we need one more issue for this since the issue is already visible on the WG project board https://github.com/orgs/cncf/projects/10/views/12 ? It has also been added as an agenda point for the Green Reviews WG meeting last week.

**Comment by nikimanoledaki** [comment]
> @nikimanoledaki @AntonioDiTuri — did we already placed this issue in wg-green-reviews with a call for contribution? Perhaps someone would like to support @immavalls writing it.

@leonardpahlke - shared a callout on the channel now!

@guidemetothemoon added the issue to the WG board + mentioned it during the last WG meeting as well 👍 I mentioned in my earlier comment that I can meet with Imma to go through it and/or review it anytime.

@immavalls I added a point to [the agenda for the WG meeting next week](https://docs.google.com/document/d/19xUur8RjnNwO2rwLw5Wwf7ISQ9V41sr0UaartIJoQvE/edit#heading=h.v4eu1k9rrsqe) so that we can go over it, answer any questions, add content, etc.

**Comment by AntonioDiTuri** [comment]
@immavalls I am up for helping in writing the article, I will give my inputs soon :)

**Comment by immavalls** [comment]
Thanks @AntonioDiTuri 🙌

**Comment by savitharaghunathan** [comment]
@immavalls @AntonioDiTuri thank you for working on this blogpost. My one suggestion is to change `cloud-native` to `cloud native` as cncf prefers the latter style. I noted it in the doc but not on all the occurrences, thats why mentioning it here.

**Comment by immavalls** [comment]
Thanks @savitharaghunathan I replaced all the occurrences! And we'll keep an eye, Grammarly suggests cloud-native and I might fall for that again

**Comment by savitharaghunathan** [comment]
@immavalls @AntonioDiTuri Just checking in on the progress. Are we on target to get this published before KubeCon? Do you need any help from the leads or co-chairs? happy to support in any way I can.

**Comment by immavalls** [comment]
Thanks for checking in @savitharaghunathan. From my side, we are okay. I did the intro and closing and reviewed the comments. Open to additional comments if anyone else has the time to read it. Once we are ok with the draft, @AntonioDiTuri wrote the progress update, I can open the PR for the blog. I'm not sure if we want to wait until next week as this is on the agenda for the WG meeting @nikimanoledaki ?

**Comment by AntonioDiTuri** [comment]
I reviewed the comments related to the project update and modified accordingly. Is there anyone we could refer to to have a WG-logo? Also if anyone has ideas for the picture in that section I am open to suggestions :)

**Comment by guidemetothemoon** [comment]
@AntonioDiTuri @immavalls @leonardpahlke I added some adjustments to the intro and conclusion of the blog post + blog post title and such. In addition for the mid-section of the blog post I've added  suggestions for the section and subsection titles + notes on what those sections should include and how some of the content can be moved around a bit. It's a combination of direct notes and comments. Does this help/make sense?

I didn't have enough time to review further on the mid-section but I'll see if I can find some time during the week. Don't hesitate to modify as you see fit in case you have any objections ☺️

**Comment by AntonioDiTuri** [comment]
Hi @guidemetothemoon thanks for the structure proposal! I reshaped my content and reviewed it according to your suggestions! Please Let me know if you have some other comments.

**Comment by guidemetothemoon** [comment]
@AntonioDiTuri @nikimanoledaki @leonardpahlke @immavalls @rossf7 I just did one more detailed review of the document - added minor changes directly and added some comments and suggestions to some minor improvements before it LGTM from my side. Please take a look and let me know if there's a need with a final review once all the comments are resolved😊

**Comment by nikimanoledaki** [comment]
Likewise, I left some final suggestions then LGTM from my side as well!

**Comment by leonardpahlke** [comment]
looks good.

**Comment by guidemetothemoon** [comment]
@AntonioDiTuri @immavalls all reviewers have now signed off and I think that we're good to create a PR and get the blog post published!🥳 Can one of you create a PR, please, if you have time?

Great work and effort everyone involved, thanks a lot for the time you put into making this blog post happen!💚

@savitharaghunathan FYI, the blog post is ready and we'll create a PR soon to get it published on the TAG ENV website asap, but you also mentioned that we should send it over to publish on CNCF website as well so here's a heads up 😊

**Comment by immavalls** [comment]
I can create the PR @guidemetothemoon. Is it okay to have it later today?

**Comment by guidemetothemoon** [comment]
Yes, sounds great, @immavalls, thanks a lot!

**Comment by savitharaghunathan** [comment]
> @savitharaghunathan FYI, the blog post is ready and we'll create a PR soon to get it published on the TAG ENV website asap, but you also mentioned that we should send it over to publish on CNCF website as well so here's a heads up 😊

Hmmm, do we want to get it published at both the places then? @leonardpahlke
